src/middlewares/authMiddleware.js

```diff
@@ -0,0 +1,22 @@
+const jwtService = require('../services/jwtService.js');
+
+const authMiddleware = (req, res, next) => {
+  const authorization = req.headers['authorization'] || '';
+  const [, token] = authorization.split(' ');
+
+  if (!authorization || !token) {
+    return res.sendStatus(401);
+  }
+
+  const userData = jwtService.verify(token);
```

Comment:
The `jwtService.verify` function should be wrapped in a try-catch block to handle any exceptions that might occur during token verification, such as invalid or expired tokens.